app/src/androidTest/java/com/swent/assos/ReminderTest.kt

```diff
@@ -1,16 +1,16 @@
 package com.swent.assos
 
-import android.util.Log
 import androidx.activity.compose.setContent
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.google.firebase.firestore.FirebaseFirestore
 import com.swent.assos.model.data.DataCache
 import com.swent.assos.model.data.User
+import com.swent.assos.model.service.impl.localDateTimeToTimestamp
```

Comment:
This will be changed soon (these helper functions will be moved to the utils file)